clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterRegistration.java

```diff
@@ -131,12 +131,20 @@ public void execute() {
           startTask(new RegisterSchemaTask(0));
         } else {
           Microservice newMicroservice = serviceCenterClient.getMicroserviceByServiceId(serviceResponse.getServiceId());
-          dealIsSwaggerDifferent(newMicroservice);
+          if (isSwaggerDifferent(newMicroservice)){
+            if (canRewriteSwagger()){
+              LOGGER.warn("Service has already registered, but schema ids not equal. However, it will continue to register");
```

Comment:
Service has already registered, but schema ids not equal, try to register it again.